compose/service/record.go

```diff
@@ -617,6 +661,13 @@ func (svc record) create(ctx context.Context, new *types.Record) (rec *types.Rec
 		return
 	}
 
+	// store revision
+	if m.ModelConfig.Revisions.Enabled {
+		if err = svc.revisions.created(ctx, new); err != nil {
+			return
+		}
+	}
```

Comment:
We discussed doing revisions as a delta where the old value is logged and the record outlines the latest state.
What made it change?

Reply:
feels a bit less magical.